.Dislocation Motion.tmp.nlogo

```diff
@@ -9,29 +9,28 @@ atoms-own [
   vx     ; x-component of velocity vector
   vy     ; y-component of velocity vector
   mass   ; mass of atom
-  pinned? ; 0 if the atom isn't pinned in place, 1 if it is (for boundaries)
-  ex-force-applied? ; is an external force directly applied to these atoms? 0 if no, 1 if yes
+  pinned? ; False if the atom isn't pinned in place, True if it is (for boundaries)
+  ex-force-applied? ; is an external force directly applied to this atom? False if no, True if yes
 ]
 
 globals [
-  eps ; used in LJ force
-  sigma ; used in LJ force
+  eps ; used in LJ force. Well depth; measure of how strongly particles attract each other
+  sigma ; used in LJ force. Distance at which intermolecular potential between 2 particles is 0
   cutoff-dist ; each atom is influenced by its neighbors within this distance (LJ force)
   dt ; time step for the velocity verlet algorithm
-  sqrt-2-kb-over-m  ; constant
-  cone-check-dist ; each atom links with neighbors within this distance
+  sqrt-2-kb-over-m  ; constant. Used when calculating the thermal velocity. Square root of (2 * boltzmann constant / m). Is not numerically accurate for this simulation.
```

Comment:
instead of saying "not numerically accurate"  say that the units are arbitrary in the simulation, so this constant is as well